app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_progress.html

```diff
@@ -26,33 +27,33 @@ <h2>Users Progress for {{ reader_study.title }}</h2>
 
     <ul class="list-group list-group-flush">
         {% for user in users %}
-            <li class="list-group-item">
+            <li class="list-group-item mb-1">
                 <div class="row">
-                    <div class="col-3 mb-1">
+                    <div class="col-6 col-lg-2 col-md-3 p-1">
                         <div class="h-100 d-flex justify-content-start align-items-center">
                             {{ user.obj|user_profile_link }}
                         </div>
                     </div>
-                    <div class="col-4 mb-1">
-                        <div class="h-100 d-flex align-items-center">
-                            <div class="flex-fill progress h-100">
-                                <div class="progress-bar" role="progressbar"
-                                     style="width: {{ user.progress.questions }}%"
-                                     aria-valuenow="{{ user.progress.questions }}"
-                                     aria-valuemin="0" aria-valuemax="100">
-                                </div>
+                    <div class="col-6 col-lg-4 col-md-3 p-1">
+                        <div class="progress h-100">
+                            <div class="progress-bar" role="progressbar"
+                                 style="width: {{ user.progress.questions }}%"
+                                 aria-valuenow="{{ user.progress.questions }}"
+                                 aria-valuemin="0" aria-valuemax="100">
+                                {{ user.progress.questions|floatformat }}%
```

Comment:
Why did you move this? We had lots of issues with that before.

Reply:
Doesn't it disappear at some point? When the value approaches 100%

Reply:
It disppeared on small screens but that was because of how the divs were set up. I fixed that and I can see it on all screen sizes with a value of 100% and 50%. I can play around some more to make sure it works, but if you prefer moving it back to how it was, also fine with me. 
Large screen
![image](https://user-images.githubusercontent.com/30069334/170031052-af1b42d7-d175-4bd7-9a54-664c976da7f7.png)
Small screen (not particularly pretty but I presume this screen isn't used very often?)
![image](https://user-images.githubusercontent.com/30069334/170031133-a00c4886-1dab-41b9-b03a-96038c3b5611.png)

Reply:
I don't recall what the exact issue was, but I think it wasn't at 100% but rather something in the 85-95 range. Maybe check that and 0 just to be sure.

Reply:
Ah I see that the text partially disappears in cases of 5% (the too big to display on the blue bar). I'll see if I can fix that and otherwise revert to how it was.

Reply:
Changing the position of the text to `absolute` and adding a shadow solves the issue: 
![image](https://user-images.githubusercontent.com/30069334/170034547-4091313f-94ca-436a-a7f4-da6f633dd215.png)

I guess it makes sense that the default behaviour is that the text disappears. It's supposed to label only the blue part... So what do you think?

Reply:
I like what you did, it's better than no text imo.

Reply:
Well, we could revert to what we did before, which is printing the progress text in a seperate column next to the progress bar.

Reply:
I would say keep it as a separate column, then the numbers are aligned when you have multiple readers with different progressions, looks tidier.